roles/common/tasks/monitoring.yml

```diff
@@ -207,6 +207,13 @@
 - name: log scanning caching dir
   file: path=/var/cache/check-log owner=sensu state=directory
 
+- name: bad memory alert
+  sensu_check: name=bad-memory plugin=check-mcelog.sh use_sudo=true
+               auto_resolve=false interval=20 occurrences=1
+               args='--criticality'
```

Comment:
I think the intent is that a value would come along with the --criticality. For this PR you can avoid sending that argument, we just want the check itself to accept such an argument if it were provided.